pages/en/exchange-operators.mdx

```diff
@@ -0,0 +1,194 @@
+import Page from "@reason/pages/Docs";
+import DocLink from "@reason/components/DocLink";
+export default Page({ title: "Exchange Operators" });
+
+# Exchange Operators
+
+Frequently asked questions about listing Mina
+
+If you are an exchange operator looking to custody MINA on behalf of users, we recommend reviewing the following frequently asked questions (FAQ) from exchanges.
+
+<Alert kind="tip">
+
+<p>We <b>highly recommend</b> using Mina’s implementation of Rosetta API to integrate Mina blockchain with your exchange. Learn how to access Rosetta API endpoints on Mina Devnet and Mainnet here (coming soon).</p>
+
+</Alert>
+
+## Basics
+
+### Where can I find third-party audit reports for Mina? 
+
+The latest third-party audit reports are publicly available here:
+- [https://research.nccgroup.com/2020/05/13/public-report-coda-cryptographic-review](https://research.nccgroup.com/2020/05/13/public-report-coda-cryptographic-review/)
+- [https://leastauthority.com/blog/audit-of-mina-ledger-application-for-o1-labs](https://leastauthority.com/blog/audit-of-mina-ledger-application-for-o1-labs/)
+- [https://research.nccgroup.com/2022/02/22/public-report-o1-labs-mina-client-sdk-signature-library-and-base-components-cryptography-and-implementation-review](https://research.nccgroup.com/2022/02/22/public-report-o1-labs-mina-client-sdk-signature-library-and-base-components-cryptography-and-implementation-review)
+
+<br></br> 
+
+<Alert kind="note">
+
+Any news and updates related to exchange listing shared by the Mina Foundation can be found on [www.minaprotocol.com](https://minaprotocol.com) or on the [official Twitter account](https://twitter.com/MinaProtocol). Mina Foundation can not individually answer any listing questions.
```

Comment:
Could we rephrase this? My concern is exchange operators may interpret it as being directed at them, whereas I think the intention is toward interested community members.


> Any news related to exchange listings of MINA will be shared on [www.minaprotocol.com/blog](https://minaprotocol.com/blog) or on the [official Twitter account](https://twitter.com/MinaProtocol). Mina Foundation can not individually answer any such questions.

Reply:
In that case, I'll remove this for now to avoid confusion. If we notice lots of questions about this or lots of non-exchanges going to this page, then we can add it.